QStore4S/src/main/java/edu/asu/qstore4s/service/impl/Statistics.java

```diff
@@ -0,0 +1,55 @@
+package edu.asu.qstore4s.service.impl;
+
+import java.util.HashMap;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.neo4j.repository.GraphRepository;
+import org.springframework.stereotype.Service;
+
+import edu.asu.qstore4s.domain.events.impl.AppellationEvent;
+import edu.asu.qstore4s.domain.events.impl.CreationEvent;
+import edu.asu.qstore4s.exception.InvalidDataException;
+import edu.asu.qstore4s.repository.AppellationEventRepository;
+import edu.asu.qstore4s.repository.ElementNeo4jConnector;
+import edu.asu.qstore4s.repository.IElementNeo4jConnector;
+import edu.asu.qstore4s.repository.RelationEventRepository;
+import edu.asu.qstore4s.service.IStatistics;
+
+/**
+ * A Service to get the statistics from QStore
+ * 
+ * @author Nikhil Aourpally, Rachita Satyasi.
+ */
+@Service
+public class Statistics implements IStatistics {
+
+    @Autowired
+    private AppellationEventRepository appeventrep;
+
+    @Autowired
+    private RelationEventRepository releventrep;
+
+    @Autowired
+    private IElementNeo4jConnector connector;
+
+    /**
+     * The method returns the statistics fetched from the database in a HashMap
```

Comment:
Try to think about what you would want to know when using this method. You can see that the method returns a HashMap by looking at the method signature, but you won't know what the keys and values are. This is what you should put here.